server/db/connection.js

```diff
@@ -15,13 +18,88 @@ class Db {
   connectDb() {
     let dbOptions = {
       dbName: "" + dbName,
-    }
-    mongoose.connect(uri, dbOptions)
+    };
+    mongoose.connect(uri, dbOptions);
+    console.log("Connected to MongoDB");
   }
 
   disconnectDb() {
-    mongoose.connection.close()
+    mongoose.connection.close();
+  }
+
+  async getAll(collectionSchema) {
+    const collection = collectionSchema.find();
+    return collection;
+  }
+
+  // CRUD Functions
+  async addUser(userFname, userLname, userRole, userId, userPw) {
+    const user = await User.create({
+      fname: userFname,
+      lname: userLname,
+      role: userRole,
+      user_id: userId,
+      pw: userPw,
```

Comment:
I don't think you need to add the `user` in front of each variable name, non blocking